tests/group_functions/group_shift.h

```diff
@@ -2,7 +2,7 @@
 //
 //  SYCL 2020 Conformance Test Suite
 //
-//  Copyright (c) 2023 The Khronos Group Inc.
+//  Copyright (c) 2024 The Khronos Group Inc.
```

Comment:
Unrelated change?

Reply:
Because the test files have the same names, it just seems that I changed the year in the wrong `group_shift.h`. It has been addressed.